mocktracer/src/dynamic_load.cpp

```diff
@@ -4,7 +4,7 @@
 #include <cstring>
 #include <exception>
 
-int OpenTracingMakeTracerFactory(const char* opentracing_version,
+int OpenTracingMakeTracerFactoryFct(const char* opentracing_version,
```

Comment:
Can we make `OpenTracingMakeTracerFactoryFct` static?

Reply:
Sure.